modules/core/src/v2/wallet.ts

```diff
@@ -513,7 +513,7 @@ export class Wallet {
       'lastLedgerSequence', 'ledgerSequenceDelta', 'maxFee', 'maxFeeRate', 'maxValue', 'memo', 'transferId', 'message', 'minConfirms',
       'minValue', 'noSplitChange', 'numBlocks', 'recipients', 'reservation', 'sequenceId', 'strategy',
       'targetWalletUnspents', 'trustlines', 'type', 'unspents', 'nonParticipation', 'validFromBlock', 'validToBlock', 'messageKey',
-      'stakingOptions', 'eip1559',
+      'stakingOptions', 'eip1559', 'keyregTxBase64',
```

Comment:
why are we whitelisting this param? I don't see this anywhere else in the SDK code base. Where is this created/used?

Reply:
That is a valid parameter in the wallet platform: https://github.com/BitGo/bitgo-microservices/blob/e201919bccd3852f9e1888c780dfcbafc4e677e4/packages/wallet-platform/app/controllers/api/v2/transactions.js#L111

I'm guessing this was passed as part in bitgo-admin, and we are now passing it in the SDK.